modelplugin/aether-2.0.0/README.md

```diff
@@ -0,0 +1,5 @@
+# YANG files
+The YANG files to accompany this plugin have been moved to 
+
+* ../../../sdran-helm-charts/config-models/aether-3.x/files/yang
```

Comment:
isn't this ../../..//roc-helm-charts/config-models/aether-2.0.x/files/yang ?

Reply:
yeah, I'll update the readme.